crates/forge_analyzer/src/definitions.rs

```diff
@@ -1540,10 +1540,15 @@ impl Visit for LocalDefiner<'_> {
     fn visit_fn_decl(&mut self, _: &FnDecl) {}
 }
 
-impl Visit for FunctionCollector<'_> {
-    fn visit_function(&mut self, n: &Function) {
-        n.visit_children_with(self);
+impl FunctionCollector<'_> {
+    fn handle_function(&mut self, n: &Function, owner: Option<DefId>) {
         let owner = self.parent.unwrap_or_else(|| {
+            if let Some(defid) = owner {
+                return defid;
+            }
+            if let Some(defid) = self.res.default_export(self.module) {
```

Comment:
Won't this alias closures in modules with a default export? You shouldn't need to check this case here, since the owner should be set to the default export if this function is called on an assign expression to `exports` or `module.exports`.